trackbasket_be/models/krogerservice.py

```diff
@@ -47,15 +47,17 @@ def closest_store(cls, zipcode):
 #       response = requests.get('https://api.kroger.com/v1/locations?', params=parameters, headers=headers)
     # return {'response': response.text, 'response1': response1.text, 'status_code': response.status_code, 'token': token, 'content_type': response.headers['Content-Type'] }  
     parsed_response = response.json()
-
+    
     if parsed_response['data'] == []:
       return  { 'error': 'no store found for this zipcode' }
 
     blacklist = ['shell company']
     i = 0
     while (parsed_response['data'][i]['name'].lower() in blacklist) and (i < number_results - 1):
       i += 1
-    if (parsed_response['data'][i]['name'] in blacklist):
+    
+    
+    if (parsed_response['data'][i]['name'].lower() in blacklist):
```

Comment:
takes care of the case where all results returned belong to the blacklist

Reply:
would be awesome to see a test for this!